lib/redcloth/formatters/base.rb

```diff
@@ -37,10 +36,10 @@ def pba(opts)
       [:"padding-right", :"padding-left"].each do |a|
         opts[:style] = "#{a}:#{opts[a]}em;#{opts[:style]}" if opts[a]
       end
-      [:style, :class, :lang, :id, :colspan, :rowspan, :title, :start, :align].each do |a|
-        atts << " #{a}=\"#{ html_esc(opts[a].to_s, :html_escape_attributes) }\"" if opts[a]
+      atts = [:style, :class, :lang, :id, :colspan, :rowspan, :title, :start, :align].map do |a|
```

Comment:
I've no strong preference here - perhaps the maintainers do - but I'd probably just make the definition of `atts` in the original version mutable with `atts = ''.dup`. Much of a muchness when it comes to the number of object allocations though 🤷‍♂️